src/main/kotlin/nl/avisi/structurizr/site/generatr/site/model/SectionTitle.kt

```diff
@@ -0,0 +1,32 @@
+package nl.avisi.structurizr.site.generatr.site.model
+
+import com.structurizr.documentation.Section
+import com.vladsch.flexmark.ast.Heading
+import com.vladsch.flexmark.ast.Paragraph
+import com.vladsch.flexmark.parser.Parser
+
+private val parser = Parser.builder().build()
+private const val MAX_TITLE_LENGTH = 50
+
+fun Section.title(): String {
+    val document = parser.parse(content)
```

Comment:
You're parsing the content using a Markdown parser, regardless of what format is used (Markdown or Asciidoc). Given that we don't support Asciidoc at the moment, it would make more sense to return `"unknown document"` when the section is in Asciidoc format.

Reply:
Yes, done.